test/unittests/compiler/riscv64/instruction-selector-riscv64-unittest.cc

```diff
@@ -379,88 +379,6 @@ TEST_P(InstructionSelectorShiftTest, Immediate) {
 INSTANTIATE_TEST_SUITE_P(InstructionSelectorTest, InstructionSelectorShiftTest,
                          ::testing::ValuesIn(kShiftInstructions));
 
-TEST_F(InstructionSelectorTest, Word32ShrWithWord32AndWithImmediate) {
```

Comment:
`Word32ShrWithWord32AndWithImmediate` tests instruction selection for `kRiscvExt32`. Since we have removed `kRiscvExt32` altogether, this test is unnecessary.